twoliter/embedded/rpm2kit

```diff
@@ -30,3 +35,90 @@ done
 
 createrepo_c "${KIT_DIR}"
 dnf --disablerepo '*' --repofrompath "kit,file:///${KIT_DIR}" repoquery --all
+
+WORK_DIR=$(mktemp -d)
+TIMESTAMP=$(date +"%FT%T.%NZ")
```

Comment:
please quote all command substitutions:
```suggestion
WORK_DIR="$(mktemp -d)"
TIMESTAMP="$(date +"%FT%T.%NZ")"
```

(not flagging the others but there are a few more)